healthtools_ke_api/views/sms_handler.py

```diff
@@ -233,6 +232,44 @@ def parse_elastic_search_results(response):
     return result_list
 
 
+def check_message(msg):
+    # check the message and if query wasn't understood, post error
+    if 'could not find' in msg:
+        print_error(msg)
+
+
+def print_error(message):
+    """
+    print error messages in the terminal
+    if slack webhook is set up, post the errors to slack
+    """
+    print("[{0}] - ".format(datetime.now().strftime("%Y-%m-%d %H:%M:%S")) + message)
+    response = None
+    if SLACK["url"]:
+        response = requests.post(
+            SLACK["url"],
+            data=json.dumps(
+                {
+                    "attachments":
+                        [
+                            {
+                                "author_name": "HealthTools API",
+                                "color": "warning",
+                                "pretext": "[SMS] Could not find a result for this SMS.",
+                                "fields": [
+                                    {
+                                        "title": "Message",
+                                        "value": message,
+                                        "short": False
+                                        }
+                                    ]
+                                }
+                            ]
+                    }),
+            headers={"Content-Type": "application/json"})
```

Comment:
I was about to let it go until I couldn't... Let's fix the indentation here to follow [style guide](https://google.github.io/styleguide/pyguide.html#Indentation).